scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/ServerJobAndTaskLogs.java

```diff
@@ -209,7 +251,15 @@ private String readFile(File file) {
     void removeLogsDirectory() {
         String logsLocation = getLogsLocation();
         logger.info("Removing logs " + logsLocation);
-        FileUtils.removeDir(new File(logsLocation));
+        while (!org.apache.commons.io.FileUtils.deleteQuietly(new File(logsLocation))) {
+            logger.warn("Could not delete folder " + logsLocation + " retrying");
+            try {
+                Thread.sleep(1000);
+            } catch (InterruptedException e) {
+                Thread.currentThread().interrupt();
```

Comment:
my mistake, InterruptedException should be catched outside the while

Reply:
ok